app/styles/utils/_mixins.scss

```diff
@@ -280,3 +283,12 @@ $prefix-for-spec:      true !default; // required for keyframe mixin
   // 2011 (IE 10)
   -ms-flex-line-pack: $value-2011;
 }
+
+@mixin hide-scrollbar {
+  scrollbar-width: none;     /* Firefox */
+  -ms-overflow-style: none;  /* Internet Explorer 10+ */
+  &::-webkit-scrollbar {     /* WebKit */
+    width: 0;
+    height: 0;
+  }
+}
```

Comment:
I think we probably don't want to do this, since it could result in usability issues for non-mobile, non-macOS users with short enough screens or windows. Scrollbar behavior is unfortunately a very complex topic, since it depends on a multitude of factors including browser, operating system, and [whether a physical mouse is plugged in to a macOS machine](https://www.nytimes.com/2017/02/02/technology/personaltech/solving-the-case-of-the-disappearing-scroll-bars.html).

Instead, I think it's better to just leave these properties alone and instead just use `[overflow: auto](https://developer.mozilla.org/en-US/docs/Web/CSS/overflow#values)`, which generally only shows the scroll bar when the content within the element is larger than the size of the element. This at least ensures that users understand content is overflowing while still hiding the scroll bar when the content fits within the element.

Reply:
Good point. Accessibility and usability are key especially for a service like ours. Removed.